core/src/main/java/bio/terra/pearl/core/service/export/EnrolleeImportService.java

```diff
@@ -151,6 +167,22 @@ public Import importEnrollees(String portalShortcode, String studyShortcode, Stu
         return dataImport;
     }
 
+    private void doTimeShifts(Map<String, String> enrolleeMap, Enrollee enrollee) {
+        //update createAt and lastUpdated
+        if (enrolleeMap.get("enrollee.createdAt") != null) {
```

Comment:
Let's chat -- I don't understand why you still need to hardcode these columns and can't do the timeshifting in the methods below

Reply:
Just tried it myself -- the issue with the code I gave you was enrollee.getId() was null since that's just the enrollee from the map.   Try the code below:
```
if (enrollee.getCreatedAt() != null) {
            timeShiftPopulateDao.changeEnrolleeCreationTime(response.getEnrollee().getId(), enrollee.getCreatedAt());
        }
```